frontend/src/__tests__/cypress/cypress/tests/e2e/dataScienceProjects/testLaunchStandaloneNotebook.cy.ts

```diff
@@ -0,0 +1,70 @@
+import { HTPASSWD_CLUSTER_ADMIN_USER } from '~/__tests__/cypress/cypress/utils/e2eUsers';
+import { projectListPage } from '~/__tests__/cypress/cypress/pages/projects';
+import { notebookServer } from '~/__tests__/cypress/cypress/pages/notebookServer';
+import { waitForPodReady } from '~/__tests__/cypress/cypress/utils/oc_commands/baseCommands';
+
+const ocDeleteNotebook = `oc -n opendatahub get notebook -o name | grep jupyter-nb | xargs oc -n opendatahub delete --ignore-not-found`;
```

Comment:
Can this be moved into utils and also paramaterised so that we can pass arguments into the function for use in other tests?

Reply:
added new util